src/signals/settings/roles/components/RolesList/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@datapunt/asc-core';
+
+import LoadingIndicator from 'shared/components/LoadingIndicator';
+import ListComponent from 'components/List';
+
+import PageHeader from 'signals/settings/components/PageHeader';
+import formatRoles from '../../services/formatRoles';
+
+const StyledListComponent = styled(ListComponent)`
+  tr:nth-child(1),
+  td:nth-child(1) {
+    width: 20%;
+  }
+`;
+
+export const RolesList = ({
+  list,
+  loading,
+}) => (
+  <div>
```

Comment:
Is the `<div>` needed or can you use a `Fragment` here instead?

Reply:
kan Fragment zijn gedaan